src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -61,14 +65,14 @@ internal static SafeSslHandle AllocateSslContext(SslProtocols protocols, SafeX50
 
                 if (!Interop.Ssl.Tls13Supported)
                 {
-                    if (protocols != SslProtocols.None &&
+                    if (sslAuthenticationOptions.EnabledSslProtocols != SslProtocols.None &&
```

Comment:
Why did this change?

Reply:
I think this was leftover from the refactoring. I refactor the code to get the last CipherSuitesPolicy test passing.